go/src/koding/kites/kloud/contexthelper/publickeys/publickeys.go

```diff
@@ -17,6 +23,17 @@ type Keys struct {
 	KeyName    string
 }
 
+func (k *Keys) Thumbprint() (string, error) {
```

Comment:
Thanks for the link @gokmen TIL `thumbprint algorithm`

I was going to add another comment if this is fingerprinting hashing algo needs to be changed , because SSH Public Key File Format [RFC]( http://www.ietf.org/rfc/rfc4716.txt) _page 5 section 4_ states hashing algorithm to be MD5 not sha1 

Also one of the comments ( same guy with the author of the response ) states thumbprinting is used for root CAs where we dont have a notion of validating http://security.stackexchange.com/questions/71199/what-is-the-difference-between-a-thumbprint-algorithm-signature-algorithm-an/71203#comment208102_71203